apps.yaml

```
@@ -1,32 +1,13 @@
-aiidalab-qe:
-  categories:
-  - technology-aiida
-  - technology-aiidalab
-  - quantum
-  git_url: https://github.com/aiidalab/aiidalab-qe.git
+my-big-map-app:
   metadata:
-    authors:
-      $ref: https://aiidalab.github.io/aiidalab-qe/metadata.json#authors
+    title: DFT_VASP
```

```suggestion
    title: DFT-VASP
```
Since this is a human-readable title, I suggest we avoid the underscore and use a hyphen instead.